docs/private-dns/connect-devices/routers/mikrotik.md

```diff
@@ -45,16 +45,20 @@ MikroTik routers use the open source RouterOS operating system, which provides r
 
 ## Use your router admin panel
 
-Use these instructions if your Keenetic router does not support DNS-over-HTTPS or DNS-over-TLS configuration:
+Use these instructions if your Mikrotic router does not support DNS-over-HTTPS configuration:
 
-1. Open the router admin panel. It can be accessed at `192.168.1.1` or `192.168.0.1`.
-1. Enter the administrator username (usually, it’s admin) and router password.
-1. Open *Webfig* → *IP* → *DNS*.
-1. Select *Servers* and enter one of the following DNS server addresses.
-    - IPv4: `94.140.14.49` and `94.140.14.59`
-    - IPv6: `2a10:50c0:0:0:0:0:ded:ff` and `2a10:50c0:0:0:0:0:dad:ff`
-1. Save the settings.
-1. Link your IP (or your dedicated IP if you have a Team subscription).
-
-- [Dedicated IPs](/private-dns/connect-devices/other-options/dedicated-ip.md)
-- [Linked IPs](/private-dns/connect-devices/other-options/linked-ip.md)
+1. Access your MikroTik router:
+    - Open your web browser and go to your router's IP address (usually `192.168.88.1`)
+    - Alternatively, you can use Winbox to connect to your MikroTik router
+    - Enter your administrator username and password
+1. Configure Plain DNS:
+    - Go to *IP* → *DNS*
+    - In the *Servers* section, add the following AdGuard DNS servers:
+        - IPv4: `94.140.14.49` and `94.140.14.59`
+        - IPv6: `2a10:50c0:0:0:0:0:ded:ff` and `2a10:50c0:0:0:0:0:dad:ff`
```

Comment:
I think you can add mention about dedicated IPv6 support that can we obtained via control panel